src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -7,8 +7,7 @@ var addCommentHandler = function(e) {
     var cancelButton = $(this).next("input[value='Cancel']");
     var formObject = $(this).parent().parent();
     var addFormRow = $(this).parent().parent().parent();
-    var panelHeading = $(this).parent().parent().parent().parent()
-        .parent().parent().parent().parent().parent().parent().prev();
+    var panelHeading = $(this).parents("[id^='panel_display-']").find("[class*='panel-heading']").first();
```

Comment:
I believe you could simply put `find(".panel-heading")` instead of `find("[class*='panel-heading']")`, it feels more concise to me.